saleor/order/__init__.py

```diff
@@ -109,11 +109,11 @@ class OrderEvents:
     PAYMENT_REFUNDED = "payment_refunded"
     PAYMENT_VOIDED = "payment_voided"
     PAYMENT_FAILED = "payment_failed"
-    PAYMENT_EVENT = "payment_event"
 
-    PAYMENT_CAPTURE_REQUESTED = "payment_capture_requested"
-    PAYMENT_REFUND_REQUESTED = "payment_refund_requested"
-    PAYMENT_VOID_REQUESTED = "payment_void_requested"
+    TRANSACTION_EVENT = "transaction_event"
+    TRANSACTION_CAPTURE_REQUESTED = "transaction_capture_requested"
+    TRANSACTION_REFUND_REQUESTED = "transaction_refund_requested"
+    TRANSACTION_VOID_REQUESTED = "transaction_void_requested"
```

Comment:
Should we create data migration for that change? 
What should be done when I have old data in the database.

Reply:
The migration is already created: see file `saleor/order/migrations/0134_alter_orderevent_type.py`

> What should be done when I have old data in the database.
Nothing. It is a new type of order event. Previous flow work normally